docs/design/ovs-pipeline.md

```diff
@@ -929,12 +1135,53 @@ The first flow outputs all unicast packets to the correct port (the port was
 resolved by the "dmac" table, [L2ForwardingCalcTable]). IP packets for which
 [L2ForwardingCalcTable] did not set bit 16 of NXM_NX_REG0 will be dropped.
 
+## Tables (AntreaProxy is disabled)
+
+![OVS pipeline](../assets/ovs-pipeline.svg)
```

Comment:
Could you rename DnatTable to DNATTable in the SVG?

Reply:
Sure, updated, @antoninbas could you please review the updated SVG file? Thanks!